source/debug.scss

```diff
@@ -0,0 +1,38 @@
+/*
+
+	HTML DEBUGGER
+	This should be used to detect tags or attributes
+	no longer supported in HTML 5.
+
+	-------------------------------------------------
+
+	This should NOT end up in the production code.
+	
+	-------------------------------------------------
+	
+	Red Border: Unsupported tags.
+	Orange Border: Not supported by all browsers.
+	Black Dashed Bottom Border: Unsupported attribute.
+	Yellow Background: Empty attributes.
+	
+*/
+
+/* HTML 5 Unsupported Tags */
+acronym, applet, basefont, big, center, dir, font, frame, frameset, menu, noframes, strike, tt
+{ border: 5px solid red !important; }
+	
+/* HTML 5 Elements that most browsers do not support. */
+bdi, command, details, dialog, summary
+{ border: 5px solid orange !important;}
+	
```

Comment:
Each selector in a comma sequence should be on its own single line<br>Line contains trailing whitespace